algebra/src/bytes.rs

```diff
@@ -282,14 +350,14 @@ impl<'a, T: 'a + ToBytes> ToBytes for &'a T {
     }
 }
 
-impl FromBytes for Vec<u8> {
-    #[inline]
-    fn read<R: Read>(mut reader: R) -> IoResult<Self> {
-        let mut buf = Vec::new();
-        let _ = reader.read_to_end(&mut buf)?;
-        Ok(buf)
-    }
-}
+// impl FromBytes for Vec<u8> {
```

Comment:
This is also weird, but let's uncomment it and forget about it

Reply:
This part was uncommented back